### ISSUE TITLE
Have holidays selectable 

### ISSUE THREAD

**Issue by PrayerfulDrop** [bruxy70/Garbage-Collection#183]
I am using the “move collection day to next day if holiday in the week.”  The issue is in the US there are many “holidays” that are not considered valid holidays for delaying trash pickup. Would like to select which holidays are valid for the “move collection day.”

Example: Veteran’s Day (Nov 10th) though is a holiday is not a holiday that changes trash collection schedule.

**Comment by bruxy70** [comment]
Added to 3.03-beta. Please test it ane let me know. Thanks.

**Comment by jasonalsing** [comment]
I do like this feature but it seems the fix that was implemented was the inverse? Example my service provider says they shift pickup on 6 holidays, it would be easier to add those holidays to an 'include' list than to try to figure out which holidays should be 'excluded'

just my thoughts.

**Comment by bruxy70** [comment]
The holidays feature is so complicated. Everyone has some opinion, and all of them are different. From what I gathered the current way is the way to go. Also supported by the current inolementation in the holiday library, where you can pop out individual holidays. But if you want to include new, you have to give the date... Which you can do by the include date feature.

**Comment by jasonalsing** [comment]
Yes you are right, holiday feature is quite complicated.  The problem with the "include_date" that I'm having, for instance, is that it takes precedence over "even weeks" setting, such that it is telling me I have recycling next week since I have an include_date for Labor Day shift that I copied over from my "weekly" trash schedule, but since it's an even week I actually DON'T have recycling. Now I can remove the include_date of course but then I have to check every year which holidays are on even weeks or not if I should shift them. Having a select based holiday list which works in conjunction with "even weeks" or "every nth week" would be nice.
